extensions/blocks/videopress/editor.scss

```diff
@@ -0,0 +1,20 @@
+.no-videopress-media-placeholder {
+	.components-placeholder__fieldset {
+		flex-direction: row-reverse;
+		align-items: flex-start;
+		justify-content: flex-end;
+
+		button {
+			display: none;
```

Comment:
It would be nice if we could prevent these from showing, rather than hiding them, but I couldn't find a way...

Reply:
Yes, not much can be done here without core support for stopping them from rendering